docs/test-results/viewing-test-results.md

```diff
@@ -44,41 +41,173 @@ You can filter live test results by owner. If the **Enable Job Visibility Across
 | Status | Filter builds by: <ul><li>Success</li><li>Failed</li><li>Complete (the test completed but was not assigned a Pass/Fail status)</li><li>Running</li><li>Error</li></ul> |
 
+## Archived Test Results
+
+The **Archive** page allows you to manage historical information about your tests and builds. It contains all the tests and builds you have run since you opened your Sauce Labs account, and new tests and builds are automatically added to it. The dashboard only displays the last 50 builds or the last 25 tests you have run.
+
+You can filter your archived test results, apply tags, create structured searches using multiple filters, and search using freeform text. You can save your search queries as favorites to simplify your future filtering.
+
+### Changing the Archive Page Layout
+
+You can change the layout of the **Archive** page by modifying the displayed columns and sorting the content in those columns.
+
+1. On the **Archive** page, click **Show Columns**.
+2. Select or deselect the columns you want to display.
+   <img src={useBaseUrl('img/test-results/test-results-archive-columns-updated.png')} alt="Archive page columns" width="400"/>
+
+3. Click **Sort By** to change the display of your results based on search score or ascending or descending dates.
+   <img src={useBaseUrl('img/test-results/test-results-archive-sort-updated.png')} alt="Archive page sorting" width="350"/>
+
+Your layout changes will be saved until you change them again.
+
+### Filtering Test and Builds Results
+
+The **Archive** page includes several filters you can use to search through your tests and builds. You can search using a single filter or multiple filters to build a structured search.
+
+1. In Sauce Labs, click **Automated** in the left panel, and then click **Archive**.
+   <img src={useBaseUrl('img/test-results/test-results-archive-nav-updated.png')} alt="Test results archive navigation" width="650"/>
+
+2. On the **Archive** page, select either **Virtual Cloud** or **Real Devices** to view the relevant device results.
+
+3. Click a filter and select an option from the dropdown or enter a term to search for.
+   <img src={useBaseUrl('img/test-results/test-results-archive-filter.png')} alt="Test results archive search filters" width="650"/>
+
+4. Click **Apply**.
+
+#### Special Filter Criteria and Operators
+
+Some filters include additional options and the ability to use special operators. See [Filters and Examples](#filters-and-examples) for more information.
+
+### Searching Test and Build Results
+
+You have two options for creating structured searches.
+
+#### Building Structured Searches from Filters
+
+With this option, you select filters and filter criteria as you would for basic filtering. Each time you make a selection, it will be added to the Search field to build the query. Using this option, there is an implied AND between the filters you select and an implied OR between the values within a specific filter.
```

Comment:
```suggestion
With this option, you select filters and filter criteria as you would for basic filtering. Each time you make a selection, it will be added to the Search field to build the query. Using this option, there is an implied AND between the filters you select and an implied OR between the values in a specific filter.
```